docs/systemd.md

```diff
@@ -0,0 +1,55 @@
+---
+id: systemd
+title: systemd configuration
+---
+
+To manage starting, logging, etc. of Gotify, systemd can be used.
+
+# Installation
+
+For this example, Gotify is installed by root in a directory called
+`/opt/gotify/` and the executable in there is called `gotify`. The config file
+will be `/etc/gotify/config.yml` using relative paths as in the example and the
+service will be run as root. A file called `/opt/gotify.service` should be
+created by root containing:
+
+```desktop
+[Unit]
+Description=Gotify
+Requires=network.target
+After=network.target
+
+[Service]
+Type=simple
+User=root
+WorkingDirectory=/opt/gotify
```

Comment:
It's possible to add more options to make this more secure, see https://github.com/gotify/server-aur-git/blob/master/gotify-server.service

Reply:
I have looked at those and the extra options used in https://gist.github.com/hypervtechnics/9cb28e67aea93cb9b87af5141bc3aa25 However, I don't see them used by systemd services I know. I don't feel comfortable adding them without knowing exactly why. I will ask some people I know that are more experienced in systemd and will get back on it.

Reply:
I have asked some people and they say it is okay as it is. The extra options need a lot of expertise to be used well and go beyond what is a minimal configuration. They might even risk making it worse when used incorrectly.